delivery.yaml

```diff
@@ -20,6 +20,7 @@ pipeline:
       make build.docker
   - desc: push
     cmd: |
+      git status
```

Comment:
why not `git diff` if VERSION has "dirty"?
```
echo $VERSION | grep -q dirty && git diff
```

Reply:
git diff shows only upstaged changes and I was not sure what was changed. git status gives a better overview https://github.com/zalando-incubator/cluster-lifecycle-manager/pull/544#issuecomment-998050897